src/containers/PlayScreen/index.js

```diff
@@ -3,11 +3,24 @@ import React from 'react';
 import Title from '../../components/Title';
 import Header from '../../components/Header';
 
-const PlayScreen = () => (
+const reset = () => {
+  window.location.href = '/';
+}
+
+const PlayScreen = ({ health }) => (
   <Header>
     <Title>
-      Play!
+      Tamago
+    </Title>
+    <Title>
+      { health <= 0 ? '( x _ x )' : '( ^ _ ^ )' }
```

Comment:
@LidiaYoshino you can extract this statement to a constant and then use it here :-)